src/components/ThreejsEditorModal.js

```diff
@@ -101,13 +101,25 @@ export class ThreejsEditorModal extends ModalDialog {
         onWindowResize();
     }
 
+    /**
+     Add the second directional light, so that the atom is more visible
+     */
+    addDirectionalLight() {
+        const directionalLight = new THREE.DirectionalLight();
+        directionalLight.name = "DirectionalLight_2";
+        directionalLight.position.copy(new THREE.Vector3(...settings.secondDirectionalLightPosition));
+        directionalLight.intensity = settings.secondDirectionalLightIntensity;
+        this.editor.scene.add(directionalLight);
+    }
+
     /**
      * Load the scene based on the given materials.
      */
     loadScene() {
         const loader = new THREE.ObjectLoader();
         const scene = loader.parse(materialsToThreeDSceneData(this.props.materials));
         this.editor.execute(new window.SetSceneCommand(scene));
+        this.addDirectionalLight();
```

Comment:
The rest of the code looks good, thank you. Here I have a question: where in the code do we add the first light? 

Maybe we should put `addDirectionalLight` there as well?

Reply:
The first light, i.e. properties for drawing a shadow caused by the first light, is passed from the default modeler at line 120.
So I think it's a good to put it in this position.